addons/udes_common/models/registry.py

```diff
@@ -9,15 +9,18 @@ class RegistryMeta(ABCMeta):
 
     def __init__(cls, name, bases, dict):
         super(RegistryMeta, cls).__init__(name, bases, dict)
-        policy_name = cls.name()
+        refactor_action_name = cls.name()
```

Comment:
Since this is generic now, can we change `refactor_action_name` to a neutral name? it is used in refactor but also in suggested locations.

Reply:
Good spot, missed that. Will amend. Any preference on the name? I suppose it could just be `name`?

Reply:
Have now changed this to `name`. :)